src/dsp_tools/models/connection.py

```diff
@@ -27,240 +29,260 @@ def check_for_api_error(response: requests.Response) -> None:
         )
 
 
+@dataclass
 class Connection:
     """
-    An Connection instance represents a connection to a DSP server.
-
-    Attributes
-    ----------
-
-    none (internal use attributes should not be modified/set directly)
+    A Connection instance represents a connection to a DSP server.
+
+    Attributes:
+        server: address of the server, e.g https://api.dasch.swiss
+        prefixes: ontology prefixes used
+        token: session token received by the server after login
+        log: if True, every request is written into a file
+        log_directory: directory where the log files are written
     """
 
-    _server: str
-    _prefixes: Union[dict[str, str], None]
-    _token: Union[str, None]
-    _log: bool
-
-    def __init__(self, server: str, prefixes: dict[str, str] = None):
-        """
-        Constructor requiring the server address, the user and password of DSP
-        :param server: Address of the server, e.g https://api.dasch.swiss
-        :param prefixes: Ontology prefixes used
-        """
-
-        self._server = regex.sub(r"\/$", "", server)
```

Comment:
do you have any idea why this was here?

Reply:
perhaps in the past, the final slash caused a problem? I just tried to remove it, and since the tests pass, I assume that the final slash is now okay...

Reply:
fair enough :)